packages/sso/src/saml.test.ts

```diff
@@ -28,6 +28,16 @@ import {
 import { sso, validateSAMLTimestamp } from ".";
 import { ssoClient } from "./client";
 import { DEFAULT_CLOCK_SKEW_MS } from "./constants";
+import type { OIDCConfig, SAMLConfig, SSOProvider } from "./types";
+
+type SSOProviderResponse = SSOProvider<{}> & {
+	samlConfig?: SAMLConfig;
+	oidcConfig?: OIDCConfig;
+};
+
+function asSSOProvider<T>(response: T): SSOProviderResponse {
```

Comment:
why do we need this type in the test case? should we update the API response type?

Reply:
Ahh yes !  I'll fix the API response type properly so this cast isn't needed in tests.